wpilibj/src/main/java/edu/wpi/first/wpilibj/util/WPIErrorMessages.java

```diff
@@ -0,0 +1,40 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
```

Comment:
```suggestion
/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
```

Reply:
Can't do that because of wpiformat issues. Everything is going to say 2018 for now.

Reply:
🥇